iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender_data.hpp

```diff
@@ -36,7 +36,7 @@ struct ChunkSenderData : public ChunkDistributorDataType
 
     const relative_ptr<mepoo::MemoryManager> m_memoryMgr;
     mepoo::MemoryInfo m_memoryInfo;
-    UsedChunkList<MAX_CHUNKS_ALLOCATE_PER_SENDER> m_chunksInUse;
+    UsedChunkList<ChunkDistributorDataType::ChunkDistributorDataProperties_t::MAX_CHUNKS_PER_SENDER> m_chunksInUse;
```

Comment:
same same but different

Reply:
Yeah, got it. Put it now as separate param in chunk sender/receiver data. Have a look.